.vscode/settings.json

```diff
@@ -34,10 +34,5 @@
     "python.linting.pydocstyleArgs": [
         "--convention=google"
     ],
-    "python.linting.banditEnabled": true,
```

Comment:
I think this is probable a mistake, so please remove this change. If it is intentional, please share why because this was just added.